### PR TITLE
Update hash comment to changed XxHash64 usage

### PR DESCRIPTION
**Pull request by dcousens**
As introduced in https://github.com/cloudflare/wrangler/pull/1221

### CODE REVIEW

File: src/sites/mod.rs [hunk 1]
```diff
@@ -257,10 +257,10 @@ fn generate_url_safe_path(path: &Path) -> Result<String> {
     Ok(path.to_string())
 }
 
-// Adds the SHA-256 hash of the path's file contents to the url-safe path of a file to
+// Adds the XXhash hash of the path's file contents to the url-safe path of a file to
 // generate a versioned key for the file and its contents. Returns the url-safe path prefix
 // for the key, as well as the key with hash appended.
-// e.g (sitemap.xml, sitemap.ec717eb2131fdd4fff803b851d2aa5b1dc3e0af36bc3c8c40f2095c747e80d1e.xml)
+// e.g (sitemap.xml, sitemap.ec717eb2131fdd4f.xml)
```
Comment (dcousens) [inline src/sites/mod.rs:263]:
```suggestion
// e.g (sitemap.xml, sitemap.ec717eb213.xml)
```
Actually this is truncated to 10 characters on L281

**Comment by threepointone** [comment]
This would break existing sites and churn installations. I don't think we should do this .

**Review by threepointone — CHANGES_REQUESTED** [review]
As mentioned.

**Comment by threepointone** [comment]
Oh wait, this doesn't change the implementation

**Review by threepointone — APPROVED** [review]
Thanks for this!

**Pull request merged**